src/sdg/templates/organisaties/update.html

```diff
@@ -76,8 +77,4 @@ <h3 class="form__subtitle">
     <template class="formset__template">
         {% include "organisaties/_prefix/lokatie_formset.html" %}
     </template>
-
-    <div class="form__actions">
-        <p class="form__action-help">Wijzigingen worden direct gepubliceerd</p>
-    </div>
```

Comment:
Why was this removed?

Reply:
Reason of removal is because it was part of the initial mock and it seemed to be superfluous and inconsistent after the re-design updates. It can be re-added with some minor adjustments.